snippets/multisigning.py

```diff
@@ -0,0 +1,102 @@
+"""Example of how we can multisign a transaction"""
+from xrpl.clients import JsonRpcClient
+from xrpl.core.binarycodec import encode_for_multisigning
+from xrpl.core.keypairs import sign
+from xrpl.models.requests import SubmitMultisigned
+from xrpl.models.transactions import AccountSet, Signer, SignerEntry, SignerListSet
+from xrpl.transaction import (
+    autofill,
+    safe_sign_and_autofill_transaction,
+    send_reliable_submission,
+)
+from xrpl.utils import str_to_hex
+from xrpl.wallet import generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234/")
+
+wallet1 = generate_faucet_wallet(client, debug=True)
+wallet2 = generate_faucet_wallet(client, debug=True)
+wallet_master = generate_faucet_wallet(client, debug=True)
+
+signer_entries = [
+    SignerEntry(account=wallet1.classic_address, signer_weight=2),
+    SignerEntry(account=wallet2.classic_address, signer_weight=1),
+]
+
+signer_list_set_tx = SignerListSet(
+    account=wallet_master.classic_address,
+    signer_quorum=2,
+    signer_entries=signer_entries,
+)
+signed_signer_list_set_tx = safe_sign_and_autofill_transaction(
+    signer_list_set_tx, wallet_master, client
+)
+signed_list_set_tx_response = send_reliable_submission(
+    signed_signer_list_set_tx, client
+)
+print("SignerListSet constructed successfully")
```

Comment:
We did more than just construct the transaction, and we don't know if the response is successful yet.

```suggestion
print("SignerListSet constructed and submitted, here's the response:")
```

Reply:
It could be worth it to have an explicit error check here (Since after this point if the SignerListSet transaction didn't work, everything else will be weird

e.g. 
```
if(signed_list_set_tx_response.result.engineResult != "tesSuccess"):
    print("Failed to submit the SignerListSet transaction!")
```

Reply:
(Note I don't know if I have the right extension to see the `engineResult` :P - I guessed what the path to that would be)

Reply:
I think it errors out and stops program automatically because it is send_reliable_submission